packages/clerk-js/src/ui/components/UserProfile/PasswordPage.tsx

```diff
@@ -119,6 +117,7 @@ export const PasswordPage = withCardStateProvider(() => {
             <Form.Control
               {...passwordField.props}
               minLength={6}
+              complexity={true}
```

Comment:
@panteliselef  🙃(super-nit) couldn’t this be just `complexity` with no value true? Just like required

Reply:
@raptisj I don't think that this is correct. The default value is false as this "feature" is an opt-in mechanism (we don't want all password fields to have this behaviour).

`required` is a valid html input attribute, whereas complexity is a custom prop